### DISCUSSION TITLE
New CS-Studio website

### DISCUSSION THREAD

**Discussion by willrogers** [ControlSystemStudio/phoebus#1798] [General]
A new website design has been introduced, but the content needs to be added.

* https://github.com/ControlSystemStudio/controlsystemstudio.org
* https://peaceful-jackson-f187cb.netlify.app/

Content will automatically be deployed when pull requests are merged.

We will want to transfer the https://controlsystemstudio.org domain.

**Comment by shroffk** [comment]
@will does this https://peaceful-jackson-f187cb.netlify.app/ automatically update as we change the content? is there a CI task setup for this.

**Reply by willrogers** [reply]
Yes, each push to master is built by Netlify.

**Reply by shroffk** [reply]
Do you use the eclipse js editor or do you have another IDE

**Reply by willrogers** [reply]
I use VS Code.

**Comment by willrogers** [comment]
There is already a grid layout on the docs page that could be used for the Meet the Team bit.

**Reply by shroffk** [reply]
https://github.com/ControlSystemStudio/controlsystemstudio.org/blob/fbbeabafbd805a6f3f531a41014816e3fd18bdc1/src/pages/docs.js#L52

**Comment by georgweiss** [comment]
Databrowser description and member list added in https://github.com/ControlSystemStudio/controlsystemstudio.org/pull/1

**Comment by shroffk** [comment]
![image](https://user-images.githubusercontent.com/2111304/110654442-215bc900-818c-11eb-8908-6e3210e7202e.png)

**Reply by shroffk** [reply]
new configuration

![image](https://user-images.githubusercontent.com/2111304/110815719-af04ea80-8258-11eb-947b-4bf8853beba2.png)

**Reply by shroffk** [reply]
@willrogers 
is this the setup you are using
https://docs.netlify.com/domains-https/custom-domains/multiple-domains/

**Reply by willrogers** [reply]
Well https://docs.netlify.com/domains-https/netlify-dns/dns-records/ but that is the correct documentation.

It is working correctly, we just had to wait.

**Reply by shroffk** [reply]
yes, I checked last evening and the redirect had started working
:)

**Comment by shroffk** [comment]
Help documentation:

For display builder the examples are great.
I was wondering if we could add a few more topics under the "help" documentation which match the topic covered in the Information column.

![image](https://user-images.githubusercontent.com/2111304/110798267-fa62cd00-8247-11eb-9d7d-94383cf3e248.png)

**Reply by shroffk** [reply]
the read the docs are a bit more easily accessible, I can link them in external documentation - specifically guides on how to improve screens.
@kasemir do you have any opinions on this

**Reply by kasemir** [reply]
I would be in favor of extending the content in `phoebus/app/display/editor/doc`. For now it only has very concise content to  "Install Example" and "Create new Display", but that would be a great place for adding the content that you suggest above.

**Reply by pavel-ch** [reply]
Reuse some of great Kay's training sheets somehow?

**Reply by shroffk** [reply]
dynamic, formula, rules, scripts --> @georgweiss 
macros, datasources --> @tanviash 
restructure widgets, Introductions --> @shroffk

**Reply by shroffk** [reply]
@pavel-ch we are going to try to use as much of the existing documentation as we can. You can add the links here and we will use them.
Currenlty looking at:
Phoebus examples, Old CS-Studio help content.

**Comment by shroffk** [comment]
We need to add a section to the documentation on how to create a site specific product

**Reply by kasemir** [reply]
Yes. But while how to use the display builder clearly belongs into the online help, so end users have it right there, this is more on the developer end of the target audience. The online help might have a "See here for details" link, but the actual info might make most sense near the source code, for example in the README.md for 
https://github.com/ControlSystemStudio/phoebus/tree/master/phoebus-product

**Reply by shroffk** [reply]
I am fine with linking to the readme..."see here for details" is perfect. 
The goal is to have at least a sentence in the developers section on the ability to create site specific products.

**Comment by georgweiss** [comment]
We are now ready to put phoebus.org into use. I assume we should just configure DNS to point to same target as controlsystemstudio.org, right?

**Reply by shroffk** [reply]
yes!
That would be great
